.github/workflows/ci.yml

```diff
@@ -1,41 +1,55 @@
+---
 name: Erlang CI
 
-on: [push, pull_request]
+"on": [push, pull_request]
 
 jobs:
 
   build:
 
-    runs-on: ubuntu-20.04
+    runs-on: ${{matrix.os}}
 
     strategy:
       matrix:
-        otp: ['23.3', '24.3', '25.0']
-        rebar: ['3.18.0']
+        include:
+          - otp: 23
+            rebar: 3.18
+            os: ubuntu-20.04
+          - otp: 24
+            rebar: 3.22
+            os: ubuntu-22.04
+          - otp: 25
+            rebar: 3.22
+            os: ubuntu-22.04
+          - otp: 26
+            rebar: 3.22
+            os: ubuntu-22.04
 
     steps:
-    - uses: actions/checkout@v2
-    - uses: erlef/setup-beam@v1
-      id: setup-beam
-      with:
-        otp-version: ${{matrix.otp}}
-        rebar3-version: ${{matrix.rebar}}
-    - name: Restore _build
-      uses: actions/cache@v2
-      with:
-        path: _build
-        key: _build-cache-for-os-${{runner.os}}-otp-${{steps.setup-beam.outputs.otp-version}}-rebar3-${{steps.setup-beam.outputs.rebar3-version}}-hash-${{hashFiles('rebar.lock')}}
-    - name: Restore rebar3's cache
-      uses: actions/cache@v2
-      with:
-        path: ~/.cache/rebar3
-        key: rebar3-cache-for-os-${{runner.os}}-otp-${{steps.setup-beam.outputs.otp-version}}-rebar3-${{steps.setup-beam.outputs.rebar3-version}}-hash-${{hashFiles('rebar.lock')}}
-    - name: Compile
-      run: ERL_FLAGS="-enable-feature all" rebar3 compile
-    - name: Format check
-      run: ERL_FLAGS="-enable-feature all" rebar3 format --verify
-    - name: Run tests and verifications (features not enabled)
-      run: rebar3 test
-    - name: Run tests and verifications (features enabled)
-      run: ERL_FLAGS="-enable-feature all" rebar3 test
-
+      - uses: actions/checkout@v3
+      - uses: erlef/setup-beam@v1
+        id: setup-beam
+        with:
+          otp-version: ${{matrix.otp}}
+          rebar3-version: ${{matrix.rebar}}
+      - name: Restore _build
+        uses: actions/cache@v3
+        with:
+          path: _build
+          key: _build-cache-for-os-${{runner.os}}-otp-${{steps.setup-beam.outputs.otp-version}}-rebar3-${{steps.setup-beam.outputs.rebar3-version}}-hash-${{hashFiles('rebar.lock')}}
+      - name: Restore rebar3's cache
+        uses: actions/cache@v3
+        with:
+          path: ~/.cache/rebar3
+          key: rebar3-cache-for-os-${{runner.os}}-otp-${{steps.setup-beam.outputs.otp-version}}-rebar3-${{steps.setup-beam.outputs.rebar3-version}}-hash-${{hashFiles('rebar.lock')}}
+      - name: Compile
+        run: ERL_FLAGS="-enable-feature all" rebar3 compile
+      - name: Format check
+        run: ERL_FLAGS="-enable-feature all" rebar3 format --verify
+      - name: Run tests and verifications (features not enabled)
+        run: rebar3 test
+      - name: Run tests and verifications (features enabled)
+        run: ERL_FLAGS="-enable-feature all" rebar3 test
+      - name: Build documentation
+        run: rebar3 ex_doc
+        if: ${{matrix.otp != 23}}
```

Comment:
`ex_doc` is 24+, but the goal here is to be able to publish prettier stuff to Hex.pm

Reply:
Once again… we can drop support for OTP23… and this is as good a reason for that as any.

Reply:
60743a3